### ISSUE TITLE
Handlebars chart - pagination support

### ISSUE THREAD

**Issue by jsanko9** [apache/superset#29657]
### Bug description

When enabling "SERVER PAGINATION" option in Handlebars chart, these is no impact on the chart generated.

Expected result: Pagination list and "show entries" option menu, same as for Table chart.

Each time we click on different page, I would expect to viewer re-render with updated 'data' variable.

### How to reproduce the bug

Any dataset with more than 10 rows is not properly paginated in Handlebars chart

### Superset version

master / latest-dev

### Python version

3.9

### Node version

16

### Browser

Chrome

### Checklist

- [X] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [X] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by jsanko9** [comment]
Unfortunately I'm unable to solve this on my own :(

**Comment by rusackas** [comment]
Hey there... I missed this ping, and this has gone silent for 250+ days. I've never actually _tried_ using server pagination with Handlebars, so I'm a bit out of my element here. I'm not sure if it's possible now, or would constitute some sort of feature request, but I'll try to find a time to look into it. Meanwhile, I'm not sure if we should call this a bug or move it to an "Ideas" discussion thread.

**Comment by rusackas** [comment]
It seems indeed this is still the case. Unless anyone wants to ADD support for server pagination, I would suspect that the lack of outcry here means it's not used much. I would suggest simply removing the relevant controls for that (unimplemented) feature.

**Comment by LisaHusband** [comment]
I don't think it is a good first issue

**Comment by LisaHusband** [comment]
a little complicated

**Comment by LisaHusband** [comment]
I don't think it is a good first issue. It's a little complicated. But I think it's good to simply delete it, because it's already in the control panel, but it's not so convenient to use, so this is a good first issue. 😆

**Comment by LisaHusband** [comment]
![Image](https://github.com/user-attachments/assets/ea5fce18-e8ec-4f60-b3f2-d1b50e8aaeab)
But from the picture it doesn't seem to be displayed at all, and there is no need to delete it

**Comment by jsanko9** [comment]
> ![Image](https://github.com/user-attachments/assets/ea5fce18-e8ec-4f60-b3f2-d1b50e8aaeab)
> But from the picture it doesn't seem to be displayed at all, and there is no need to delete it

The server pagination option is in your screenshot. It doesn't do anything

**Comment by LisaHusband** [comment]
Sorry, I didn't notice it due to my non-native language

**Comment by LisaHusband** [comment]
![Image](https://github.com/user-attachments/assets/c15cb2d8-5761-4434-8a4a-484c26e27d5b)
It's disappear now 😄

**Comment by jsanko9** [comment]
> ![Image](https://github.com/user-attachments/assets/c15cb2d8-5761-4434-8a4a-484c26e27d5b)
> It's disappear now 😄 

Happy and sad at the same time. But I guess it's solved now.
